wire-runtime/src/test/java/com/squareup/wire/UnknownFieldsTest.java

```diff
@@ -89,4 +90,73 @@ public void testUnknownFields() throws IOException {
     assertThat(v2C.v2_f64).isEqualTo(new Long(98765L));
     assertThat(v2C.v2_rs).containsExactly("1", "2");
   }
+
+  @Test
+  public void testBuilderAddingAndClearingUnknownFields() throws IOException {
```

Comment:
Overall I generally don’t like these tests that focus on exercise the general API and not any particular features.

Would you consider writing focused tests to cover specific inputs?

Maybe a single, smalltest-possible test to confirm the following:
 - repeated calls to `build()` retain unknown fields
 - unknown fields can be added between calls to build
 - clear unknown fields works

Ideally if a test fails it’s clear from the test name which specific behavior is broken